make-uniform/production/Build Standard Database.Rmd

```diff
@@ -387,31 +387,18 @@ survey_flat <- survey_flat %>%
-  mutate(survey_tech = ifelse(operator == "SF Muni" & str_count(route, pattern = fixed("M-Oce", ignore_case = TRUE)) > 0L, "light rail", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "SF Muni" & str_count(route, pattern = fixed("N-Jud", ignore_case = TRUE)) > 0L, "light rail", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("101", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("102", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("103", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("104", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("120", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("121", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("122", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("140", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("168", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("180", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("181", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("182", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("185", ignore_case = TRUE)) > 0L, "express bus", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("900", ignore_case = TRUE)) > 0L, "light rail", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("901", ignore_case = TRUE)) > 0L, "light rail", survey_tech)) %>%
-  mutate(survey_tech = ifelse(operator == "VTA" & str_count(route, pattern = fixed("902", ignore_case = TRUE)) > 0L, "light rail", survey_tech))
+# There was a flag in the original conversion text for VTA route 185, but no 
+# records had a route 185, so it has been dropped from the csv.
+survey_tech_update <- read.csv("bespoke_survey_tech_replacement.csv")
```

Comment:
I'd recommend explicitly reading this in as characters (`colClasses = 'character'`) given the mix of data types you're dealing with in the database.